src/main/java/com/powsybl/openloadflow/sensi/AcSensitivityAnalysis.java

```diff
@@ -174,6 +175,14 @@ public void analyse(Network network, List<PropagatedContingency> contingencies,
         Objects.requireNonNull(resultWriter);
         Objects.requireNonNull(reporter);
 
+        boolean breakers = !allSwitchesToOpen.isEmpty();
+        if (breakers && hasBusTargetVoltage(factorReader)) {
```

Comment:
@annetill Maybe to check the performance impact because we fully read factors 2 times with this. AC sensitivity analysis is quite slow so probably no impact, but to check with a larger number of factor.

Reply:
Maybe could have been done inside the actual factor read method without doing it 2 times?

Reply:
I can see that it was already the case to check for transformer voltage control. So new we read all factors and resolve to network 3 times.

Reply:
Fixed.